src/classes/Account.js

```diff
@@ -479,16 +468,16 @@ class Account {
 			.then(userID => (
 				new Promise((resolve, reject) => {
 					const { user } = conf.account;
-					if (user === null) {
+					if (!user) {
 						return this.getUser()
 							.then((u) => {
-								if (u.emailValidated !== true) {
+								if (!u.emailValidated) {
```

Comment:
Is this guaranteed to be semantically the same?

Reply:
good catch, I will rollback